.gitignore

```diff
@@ -2,4 +2,5 @@ bin/
 obj/
 /packages/
 riderModule.iml
-/_ReSharper.Caches/
\ No newline at end of file
+/_ReSharper.Caches/
+.idea
```

Comment:
I think that you should also add `.vs` over here, that folder is unnecessary in the codebase.
https://github.com/Aryan-sharma11/AddSevenPS/tree/develop/.vs/AddSevenPS/v17


You'll need to update the gitignore and then forcefully commit the removal of `.vs` folder.

Reply:
I resolved this problem in commit 676f1bcfb910d79c27c7ea4ef9a745a8c59e6327